thrifty-runtime/src/commonMain/kotlin/com/microsoft/thrifty/ThriftException.kt

```diff
@@ -30,8 +30,8 @@ import kotlin.jvm.JvmStatic
  */
 class ThriftException(
         @JvmField
-        val kind: Kind?,
-        message: String?) : RuntimeException(message) {
+        val kind: Kind,
+        message: String?) : RuntimeException(message), Struct {
```

Comment:
nice, I should have thought of this long ago :)

Reply:
Wasn't really necessary because you were never writing it :)